From 900926933ef73518d2477546b4b5d2d4ffdb787d Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 00:48:02 +0200
Subject: [PATCH 2/4] Telephony: Mark enhanced_4g_lte_mode_title_variant as
 untranslatable

Fixes:
warn: removing resource com.android.phone:string/enhanced_4g_lte_mode_title_variant without required default value.

Change-Id: Iad9e62a4eaca087f1e1df417e7de3812461cbaeb
---
 res/values-ast-rES/strings.xml | 9 ---------
 res/values-cy/strings.xml      | 9 ---------
 res/values/strings.xml         | 2 +-
 3 files changed, 1 insertion(+), 19 deletions(-)

diff --git a/res/values-ast-rES/strings.xml b/res/values-ast-rES/strings.xml
index cd06a78d9..b16ed7740 100644
--- a/res/values-ast-rES/strings.xml
+++ b/res/values-ast-rES/strings.xml
@@ -433,15 +433,6 @@
     <!-- Mobile network settings screen, name for network operator category -->
     <!-- Mobile network 4G title [CHAR LIMIT=30] -->
     <string name="enhanced_4g_lte_mode_title">Mou 4G LTE ameyoráu</string>
-    <!-- Carrier variant of Enhaced 4G LTE Mode title.  [CHAR LIMIT=50] -->
-    <string-array name="enhanced_4g_lte_mode_title_variant">
-        <!-- 0: Default -->
-        <item>@string/enhanced_4g_lte_mode_title</item>
-        <!-- 1: Verizon -->
-        <item>Llamaes avanzaes</item>
-        <!-- 2: O2 UK -->
-        <item>4G Calling</item>
-    </string-array>
     <!-- Mobile network 4G summary [CHAR LIMIT=80] -->
     <string name="enhanced_4g_lte_mode_summary">Usa los servicios LTE p\'ameyorar la voz y otres comunicaciones (aconséyase)</string>
     <!-- Mobile network settings screen, data enabling checkbox name -->
diff --git a/res/values-cy/strings.xml b/res/values-cy/strings.xml
index 4cf46c8bd..b7c40b8db 100644
--- a/res/values-cy/strings.xml
+++ b/res/values-cy/strings.xml
@@ -452,15 +452,6 @@
     <string name="network_operator_category">Rhwydwaith</string>
     <!-- Mobile network 4G title [CHAR LIMIT=30] -->
     <string name="enhanced_4g_lte_mode_title">Modd 4G LTE Gwell</string>
-    <!-- Carrier variant of Enhaced 4G LTE Mode title.  [CHAR LIMIT=50] -->
-    <string-array name="enhanced_4g_lte_mode_title_variant">
-        <!-- 0: Default -->
-        <item>@string/enhanced_4g_lte_mode_title</item>
-        <!-- 1: Verizon -->
-        <item>Galwadau Uwch</item>
-        <!-- 2: O2 UK -->
-        <item>4G Calling</item>
-    </string-array>
     <!-- Mobile network 4G summary [CHAR LIMIT=80] -->
     <string name="enhanced_4g_lte_mode_summary">Defnyddio gwasanaethau LTE i wella llais a chyfathrebu arall (argymhellir)</string>
     <!-- Mobile network settings screen, data enabling checkbox name -->
diff --git a/res/values/strings.xml b/res/values/strings.xml
index 905352095..c3939697d 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -517,7 +517,7 @@
     <!-- Mobile network 4G title [CHAR LIMIT=30] -->
     <string name="enhanced_4g_lte_mode_title">Enhanced 4G LTE Mode</string>
     <!-- Carrier variant of Enhaced 4G LTE Mode title.  [CHAR LIMIT=50] -->
-    <string-array name="enhanced_4g_lte_mode_title_variant">
+    <string-array name="enhanced_4g_lte_mode_title_variant" translatable="false">
         <!-- 0: Default -->
         <item>@string/enhanced_4g_lte_mode_title</item>
         <!-- 1: Verizon -->
-- 
2.17.1

